Design/Usage/AlgorithmUsageStatistics.md

```diff
@@ -0,0 +1,185 @@
+#Overview
+
+We have several algorithms that are similar and would love to rationalise the code and make the choice of which algorithm to use easier for scientists.
+However we always face the problem that we never know how much an algorithm is used.  It was agreed at the SSC2015 the we could track the usage of algorithms alongside simple startups of Mantid.
+
+This design document should be considered as an extension of the previous implemented design [MeasureUsageStatistics](MeasureUsageStatistics.md).
+
+#Requirements
+
+##Must
+
+1. Must record Mantid startup as is currently done.
+2. Must record algorithm usage, in a flexible way that allows ad-hoc queries to be performed.
+3. Must not significantly adversly affect Mantid interactive performance.
```

Comment:
For completeness. If we're going to say something about performance, I also think it would be worth specifying reliability (even though strictly they are both functional) as one of the must-have requirements. Basically the requirements should be that we do nothing to compromise normal operation of the algorithms or the framework. Sounds obvious but it should filter down into design considerations.